src/phantom.js

```diff
@@ -52,13 +50,11 @@ export default class Phantom {
      * @param [logger] object containing functions used for logging
      * @param [logLevel] log level to apply on the logger (if unset or default)
      */
-    constructor(args?: string[] = [],
-        {phantomPath = phantomjs.path, logger = defaultLogger, logLevel = defaultLogLevel}: Config =
-        {
-            phantomPath: phantomjs.path,
-            logger: defaultLogger,
-            logLevel: defaultLogLevel,
-        }) {
+    constructor(args?: string[] = [], {
+        phantomPath = phantomjs.path,
+        logLevel = defaultLogLevel,
+        logger = createLogger({logLevel}),
```

Comment:
... and a new one is always created if not provided, so they'll never "change logLevel" (as is being tested in the tests)